CHANGELOG.md

```diff
@@ -24,6 +24,19 @@ CHANGELOG
 Swift 5.0
 ---------
 
+* [SE-0235][]:
+
+  The standard library now contains a `Result<Success, Failure: Error>` type to manually model potentially error-producing code.
+  
+  ```swift
+  enum Result<Success, Failure: Error>
+      case success(Success)
+      case failure(Failure)
+  }
+  ```
+  
+  This type can be used in contexts where throwing or catching errors may be undesired or impossible while still using `Error`-conforming types, such as asynchronous methods or methods where Swift's automatic error propagation isn't ergonomic.
```

Comment:
I think this text could more closely reflect the guidance on the Forums surrounding when to use `Result` versus throwing.

Reply:
Could outline the points you'd like to see?

Reply:
"Undesired or impossible" is pretty vague. Some well received answers on that point--

@jckarter wrote:
> I think it's reasonable to see Result and throw as serving different needs within the general realm of error handling and propagation. As I'd mentioned in the previous thread, throw was intentionally designed with an eye toward loosely-coupled, handled-at-a-distance failures, which is why it features automatic propagation (to minimize the overhead of carrying the error to the handler) and no typing (because types would not really help you handle all uncountably many forms of network/file/hardware/etc. error that could accrete through layers of a system). A lot of the people reaching for Result or wishing for typed throws are dealing with more immediate error conditions for which implicit propagation and try/catch ceremony are overkill, and being able to be more specific about the result type is valuable to contain the set of error conditions.

You wrote:
> Result does indeed provide something new to the language: manually propagated error handing. While use of this case is not going to be as popular as the automatic try/throw, it has uses that that construct can't handle.